test/bankaccountverification/api/ApiControllerSpec.scala

```diff
@@ -101,7 +102,9 @@ class ApiControllerSpec extends AnyWordSpec with Matchers with MockitoSugar with
           address = Some(InitRequestAddress(List("Line 1", "Line 2"), Some("Town"), Some("Postcode"))),
           timeoutConfig = Some(InitRequestTimeoutConfig("url", 100, None))))
 
-        val fakeRequest = FakeRequest("POST", "/api/init").withJsonBody(json)
+        val fakeRequest = FakeRequest("POST", "/api/init")
```

Comment:
should be using api/v2/init for tests as we are going to deprecate this in the near future.

Reply:
We could but that should be done as part of the v2 ticket, unless thats already been played and this is a hangover?